WordPress/src/main/java/org/wordpress/android/models/ReaderPost.java

```diff
@@ -665,6 +665,7 @@ public boolean isSamePost(ReaderPost post) {
                && post.isLikedByCurrentUser == this.isLikedByCurrentUser
                && post.isCommentsOpen == this.isCommentsOpen
                && post.useExcerpt == this.useExcerpt
+               && post.isBookmarked == this.isBookmarked
```

Comment:
In my understanding, while `isBookmarked` is not supported by the server, the variable will always be `false` by default, so this should not affect posts that are not bookmarked.

Reply:
Correct, but when the post is bookmarked and you fetch it from the server (with isBookmarked false) it will be considered as changed. Even though the posts haven't changed.

Reply:
I removed the flag, as it was indeed causing some problems. I didn't really like the idea of refreshing Reader list every time we come back to it, so I added a  temporary method that compares bookmark flag in addition to the regular content of posts. As you mentioned, we should ideally use DIffUtil with our Recycle lists, but this is outside of the scope of this issue, so I made a new one to keep track of it #7738